src/routes/auth.ts

```diff
@@ -27,12 +26,15 @@ export const getUserPublicKeys: RequestHandler = (req, res) => {
 				},
 			},
 			passthrough: { req },
-		})
-		.then((data: Array<{ public_key: string }>) => {
-			const authorizedKeys = data.map(e => e.public_key).join('\n');
-			res.status(200).send(authorizedKeys);
-		})
-		.catch(err => {
-			handleHttpErrors(req, res, err);
```

Comment:
I also fixed a bug here where it was possible that we'd never respond to the request if the error wasn't a `HttpError`